go-minitwit/user_handlers.go

```diff
@@ -97,18 +97,23 @@ func userTimelineHandler(c *gin.Context) {
 
 	// does the logged in user follow them
 	followed := false
-	pUserId := profileUser[0]["user_id"].(int64)
-	profileName := profileUser[0]["username"]
+	pUserId := int64(profileUser.UserID)
```

Comment:
I think UserID is already an int, isn't it?

Reply:
also use normal int (less bytes = efficiency)

Reply:
ok changing